Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+  && apt-get install -y python3-pip python3-dev \
+  && cd /usr/local/bin \
+  && ln -s /usr/bin/python3 python \
+  && pip3 install --upgrade pip \
+  && apt-get install libssl-dev \
+  && apt-get install -y locales
+RUN locale-gen en_US.UTF-8
+ENV LC_ALL=en_US.UTF-8
+ENV LANG=en_US.UTF-8
+COPY requirements-docker.txt /home/
+RUN pip install -r /home/requirements-docker.txt
+RUN pip install docker-compose
+COPY .env /home/
+COPY spin-docker.py /home/
+COPY docker-compose.yml /home/
+COPY createPemFiles.py /home/
```

Comment:
Any reason why not use single COPY here?
i.e
`COPY .env spin-docker.py docker-compose.yml createPemFiles /home/`

Reply:
Not really, I can fix it, thanks